### PR TITLE
feat!: transfer gke action

### PR DESCRIPTION
**Pull request by bharathkkb**
- Transfer action from https://github.com/google-github-actions/setup-gcloud/tree/master/get-gke-credentials

### CODE REVIEW

File: .github/workflows/get-gke-credentials-it.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  gcloud:
+    name: with setup-gcloud & kubectl
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master # Set up ADC for credentials
```
Comment (averikitsch) [inline .github/workflows/get-gke-credentials-it.yml:11]: update

File: action.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Get GKE Credentials
+description: |-
+  Google Kubernetes Engine (GKE) is a secured and managed Kubernetes service with 
+  four-way auto scaling and multi-cluster support. Use this action to authenticate to
+  a GKE cluster by generating an emphemeral kubeconfig that can be used with kubectl, helm etc.
+author: GoogleCloudPlatform
```
Comment (averikitsch) [inline action.yaml:20]: nit: Google LLC

**Review by averikitsch — APPROVED** [review]
Just to note, I have skimmed these files since they have all gone through review

File: action.yaml (hunk 2 above)
Comment (bharathkkb) [reply action.yaml]: done

File: .github/workflows/get-gke-credentials-it.yml (hunk 1 above)
Comment (bharathkkb) [reply .github/workflows/get-gke-credentials-it.yml]: done

**Pull request merged**